pkg/hintrunner/zero/zerohint_dictionaries.go

```diff
@@ -59,6 +59,73 @@ func createDefaultDictNewHinter(resolver hintReferenceResolver) (hinter.Hinter,
 	return newDefaultDictNewHint(defaultValue), nil
 }
 
+// DictRead hint reads the result of given key from given dictionary
+// and writes it to value
+//
+// `newDictReadHint` takes 3 operanders as argument
+//   - `dictPtr` variable will be pointer to the dictionary to read from
+//   - `key` variable will be the key read from the dictionary
+//   - `value` variable will be the variable to which the result of
+//     the `key` read from the dictionary pointed to by `dictPtr`
+//     will be written to
```

Comment:
I think this description might be simpler.

`value` variable will hold the value of the dictionary stored at `key`